addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -1936,7 +1936,7 @@ QUnit.test('remove an uploading attachment', async function (assert) {
     );
     assert.containsOnce(
         document.body,
-        '.o_Composer .o_Attachment.o-temporary',
+        '.o_Composer .o_Attachment.o-uploading',
```

Comment:
Since diff in this LOC, can remove the `.o_Composer` part, which is useless

Reply:
Done